maps/delta/outfits/guardsmen.dm

```diff
@@ -12,15 +12,10 @@ Begin Warhammer loadouts
 	shoes = /obj/item/clothing/shoes/jackboots/cadian
 	head = /obj/item/clothing/head/helmet/guardhelmet
 	mask = /obj/item/clothing/mask/gas/half/cadianrespirator
-//	l_ear = /obj/item/device/radio/headset/red_team //Assign_random_squad on Equip() should be handing down a headset.
-//	r_ear = null
 	glasses = /obj/item/clothing/glasses/cadiangoggles
-//	id = null
 	l_pocket = /obj/item/storage/box/ifak
-//	r_pocket = null
+r_pocket = null
```

Comment:
Remember to compile your code in Dream Maker right before making a PR so you can catch any issues like this.

Reply:
Aye.